ibis/backends/flink/__init__.py

```diff
@@ -301,6 +304,36 @@ def version(self) -> str:
 
         return pyflink.version.__version__
 
+    def _register_udfs(self, expr: ir.Expr) -> None:
```

Comment:
This logic already exists under `_compile_udf` in the `SQLGlotBackend` base class. I know the name isn't really the right one, but can you reuse it here instead of reimplementing very similar logic?

Reply:
Sorry, I was looking at the wrong backend, scratch what I said.